uber/config.py

```diff
@@ -13,15 +13,9 @@ def mixin(cls, klass):
     def include_plugin_config(self, plugin_config):
         """Plugins call this method to merge their own config into the global c object."""
 
-        for attr, val in plugin_config.items():
-            if not isinstance(val, dict):
-                setattr(self, attr.upper(), val)
-
-        if 'enums' in plugin_config:
-            self.make_enums(plugin_config['enums'])
-
-        if 'dates' in plugin_config:
-            self.make_dates(plugin_config['dates'])
```

Comment:
Oof, yup, those are pretty problematic.

This started because I wanted to define a new ribbon called `dealer_asst_ribbon` - when I initially did this, I found that I now have ONLY that ribbon. Obviously that's not what we want.

Technically we could skip defining enums in plugins for now and just make sure to add it to Anthrocon's development.ini, but that would make the plugin incomplete.

Thaeli mentioned earlier taking the `c` object definition out of runtime into a separate command, something like `sep load_config_options`. This is something she needs for the config DB and it seems like it would be a good way to work around a lot of these issues. We could do something like:

1. Load the uber config
2. Set the `c` object to any variables that we don't want to be changed by plugins
3. Load plugins' config and merge them into the uber config
4. Set the rest of the options for `c`.